ui/app/js/services/releases_service.js

```diff
@@ -114,6 +114,16 @@ angular.module("app").factory('Releases', function($http, $q) {
       url += '&csrf_token=' + encodeURIComponent(csrf_token);
       return $http.delete(url);
      },
+
+    signoffOnScheduledChange: function(sc_id, data) {
+      var url = "/api/scheduled_changes/releases/" + sc_id + "/signoffs";
```

Comment:
Proposal for improvement: I'm not sure. Maybe we can extract the url generation into its own service. For instance:
```js
 angular.module("app").factory('ScheduledChanges', function() {
   var service = {
      signoffUrl: function(signoff_type, sc_id) {
           return "/api/scheduled_changes/" + signoff_type +"/"  + sc_id + "/signoffs"
      }

      revokeSignoffUrl: () => {}; // TODO implement this one
   };
   return service;
});
```

What do you think?